apps/web/src/pages/Skills/SkillPortfolioPage.tsx

```diff
@@ -139,15 +139,15 @@ const SkillPortfolio = ({ userSkills, skills }: SkillPortfolioProps) => {
               <p data-h2-margin="base(x1 0)">
                 {intl.formatMessage({
                   defaultMessage:
-                    "Your skill showcase allows you to curate lists of skills from your library that present a more targeted story about your strengths and areas of interest. While your skill library acts as a central place to manage all of the skills you add to your profile, the showcases you complete are paired with your job applications to help recruiters and managers see a more complete picture of your talent.",
-                  id: "ccR/uJ",
+                    "Your skill showcase allows you to curate lists of skills from your portfolio that present a more focused and personalized story about your strength and areas of interest. While your skills portfolio serves as a central place to manage all the skills you add to your profile, the showcase is paired with your job applications. It's shared with recruiters and managers to give them a fuller picture of your talents.",
```

Comment:
There was a typo in my issue, it should stay "strengths" with an s.

Reply:
35a937dfb1a5cfa1897f998a1fb095dc9dc33c84

## Screenshot
<img width="1127" alt="Screen Shot 2024-12-31 at 08 55 53" src="https://github.com/user-attachments/assets/35064e99-c2e0-40d4-ab7c-08cdbe452e39" />